meson.build

```diff
@@ -99,7 +99,7 @@ filter_subplugin_install_dir = join_paths(subplugin_install_prefix, 'filters')
 decoder_subplugin_install_dir = join_paths(subplugin_install_prefix, 'decoders')
 customfilter_install_dir = join_paths(subplugin_install_prefix, 'customfilters')
 converter_subplugin_install_dir = join_paths(subplugin_install_prefix, 'converters')
-unittest_install_dir = join_paths(subplugin_install_prefix, 'unittest')
+unittest_base_dir = join_paths(nnstreamer_bindir, 'unittest-nnstreamer')
```

Comment:
I think `unittest-nnstreamer` is somewhat redundant because `nnstreamer_bindir` already contains `nnstreamer`. Is there any purpose on this naming?

Reply:
Simply set directory name for nnst, nntr and ml api. The install path for ml api is 'unittest-ml'.